.github/workflows/spanner-emulator-system-tests.yaml

```diff
@@ -6,7 +6,7 @@ on:
       - 'Spanner/**'
   pull_request:
     paths:
-      - 'Spanner/**'
+      - '**'
```

Comment:
Why are we changing this?

Reply:
This was just to trigger the action to run. This will not be a part of the change.